leaflet/leaflet.d.ts

```diff
@@ -805,7 +805,7 @@ declare namespace L {
          * }
          * ```
          */
-        pointToLayer?: (geoJsonPoint: GeoJSON.Point, latlng: LatLng) => Layer; // should import GeoJSON typings
+         pointToLayer?: (geoJsonPoint: GeoJSON.Point|GeoJSON.Feature<GeoJSON.Point>, latlng: LatLng) => Layer; // should import GeoJSON typings
```

Comment:
Could you remove the extra space at the start of the line? 

I haven't used this feature, so I'm curious whether this function really accepts `GeoJSON.Point` as well or if we were wrong from the start and it only accepts `GeoJSON.Feature<GeoJSON.Point>`?

Thanks in advance.